scripts/install_prereqs.sh

```diff
@@ -20,11 +20,16 @@
 #   - METHOD - xcodebuild or cmake; default is xcodebuild
 
 case "$PROJECT-$METHOD" in
-  *-xcodebuild|*-pod-lib-lint)
+  *-xcodebuild)
     bundle install
     gem install xcpretty
     ;;
 
+  *-pod-lib-lint)
+    bundle install
+    bundle exec pod repo update
```

Comment:
Shouldn't this one line be in the `pod_install.sh` script?

If this is getting too complicated maybe we should combine `install_prereqs.sh` and `pod_install.sh`? We originally split them to allow us to have separate timing entries in the logs but otherwise there's no reason for these to be separate.